docs/sources/operators-guide/getting-started/_index.md

```diff
@@ -24,23 +27,24 @@ The written instructions focus on deploying Grafana Mimir as a [monolith]({{< re
 
 - Using Docker:
 
-```bash
-docker pull grafana/mimir:latest
-```
+  ```bash
+  docker pull grafana/mimir:latest
+  ```
 
 - Using a local binary:
 
-Download the appropriate [release asset](https://github.com/grafana/mimir/releases/latest) for your operating system and architecture and make it executable.
-For Linux with the AMD64 architecture:
+  Download the appropriate [release asset](https://github.com/grafana/mimir/releases/latest) for your operating system and architecture and make it executable.
 
-```bash
-curl -fLo mimir https://github.com/grafana/mimir/releases/latest/download/mimir-linux-amd64
-chmod +x mimir
-```
+  For Linux with the AMD64 architecture:
```

Comment:
Reviewers: This is a new line to make it more readable. No other content change.